mlir/test/Dialect/Linalg/test-fuse-consumer.mlir

```diff
@@ -0,0 +1,103 @@
+// RUN: mlir-opt %s -split-input-file -test-linalg-fuse-consumer | FileCheck %s
+
+#map = affine_map<()[s0] -> (64 ceildiv s0)>
+#map1 = affine_map<(d0)[s0] -> (d0 * s0)>
+#map2 = affine_map<(d0)[s0] -> (-(d0 * s0) + 64, s0)>
+// CHECK-LABEL: func.func @fuse_tileable_consumer
+// CHECK-SAME: %[[CHUNK_SIZE:[0-9a-z]+]]: index
```

Comment:
I tried your method, but there's an issue with matching because filecheck will match extra characters due to having the same data type on the same line, causing errors.

Reply:
I see, thanks for trying. Maybe let's update to `[0-9A-Za-z_]+` then, which should capture all supported names.